src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -146,7 +146,14 @@ class PalantirDockerPlugin implements Plugin<Project> {
     }
 
     private String computeName(String name, String tag) {
-        return name.replaceAll(":.*", "") + ":" + tag
+        def lastBackSlashIndex = name.lastIndexOf("/");
```

Comment:
This method is a little hard to understand. Needs a better name, maybe a comment.